web/themes/custom/cloudy/cloudy.libraries.yml

```diff
@@ -1,9 +1,14 @@
-global-styling:
+bootstrap:
+  # Note: the Bootstrap CSS is currently contained within the `global` library
   js:
-    dist/main.bundle.js: {}
+    dist/bootstrap.bundle.js: {}
+
+global:
+  js:
+    dist/cloudy.bundle.js: {}
   css:
     component:
-      dist/style.bundle.css: {}
+      dist/cloudy.css: {}
       https://use.fontawesome.com/releases/v5.2.0/css/all.css:
```

Comment:
FontAwesome doesn't seem to load in Drupal or Pattern Lab.

Reply:
Picking this up in the powr-1783 branch. Resolving for this branch. I'll merge both fixes to master before pushing to Production to prevent regressions.